example_agents/acme_dqn/requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Note: Reverb (the DeepMind experience database) is only available on linux,
-# as a result, this agent will only run on linux.  
-
+# as a result, this agent will only run on linux.
+git+https://github.com/agentos-project/agentos
```

Comment:
Are we sure we want this pointing to master? I guess during dev and testing, the developer could change this if necessary to point to the `test_staging` branch?

Reply:
Yeah it's not ideal.  At least the acme agent in master should always work against the rest PCS/AOS in master, so maybe this prevents some of the failure scenarios.  I think once we split AOS out from PCS and release a version of both, we can pin this to a specific version of AOS and prevent more failure scenarios.  Filed #245